test/WebJobs.Script.Tests/ScriptHostManagerTests.cs

```diff
@@ -67,8 +67,7 @@ public void RunAndBlock_DisposesOfHost_WhenExceptionIsThrown()
         {
             ScriptHostConfiguration config = new ScriptHostConfiguration()
             {
-                RootScriptPath = Environment.CurrentDirectory,
-                TraceWriter = NullTraceWriter.Instance
```

Comment:
These were old lines not needed anymore. The ScriptHost will default the TraceWriter when not set explicitly.